src/content/docs/extensions/iceberg.mdx

```diff
@@ -22,24 +22,53 @@ To test the examples, download the [iceberg_data.zip](TODO: upload iceberg sampl
 
 ## Usage
 
-### Common Scan Options
+### Common Functions
+
+#### Scan the Iceberg table
+```cypher
+LOAD FROM 'iceberg_table_path' (file_format='iceberg') RETURN *;
+```
+#### Copy the Iceberg table into a node table
+```cypher
+COPY table_name FROM 'iceberg_table_path' (file_format='iceberg');
+```
+#### Access the Iceberg Metadata
+```cypher
+CALL ICEBERG_METADATA('iceberg_table_path') RETURN *;
+```
+#### List the Iceberg Snapshots
+```cypher
+CALL ICEBERG_SNAPSHOTS('iceberg_table_path') RETURN *;
+```
+
+### Common Options
 
 ```
 ┌───────────────────────────────┬───────────┬──────────────┬──────────────────────────────────────────────────────────────────────────────┐
 │ Parameter                     │ Type      │ Default      │ Description                                                                  │
 ├───────────────────────────────┼───────────┼──────────────┼──────────────────────────────────────────────────────────────────────────────┤
-│ allow_moved_paths             │ BOOLEAN   │ false        │ Allows scanning Iceberg tables that are moved                                │
-│ metadata_compression_codec    │ STRING    │ ''           │ Treats metadata files as when set to 'gzip'                                  │
-│ version                       │ STRING    │ '?'          │ Provides an explicit version string, hint file or guessing                   │
-│ version_name_format           │ STRING    │ 'v%s%s.metadata.json,%s%s.metadata.json' │ Controls how versions are converted to metadata file names │
+│ allow_moved_paths             │ BOOLEAN   │ false        │ Allows scanning Iceberg tables that are not located in their original directory │
+│ metadata_compression_codec    │ STRING    │ ''           │ Specifies the compression code used for the metadata files                   │
+│ version                       │ STRING    │ '?'          │ Provides an explicit Iceberg version number, if not provided, the Iceberg version number would be determined from `version-hint.txt`│
+│ version_name_format           │ STRING    │ 'v%s%s.metadata.json,%s%s.metadata.json' │ Provides the regular expression to find the correct metadata data file │
 └───────────────────────────────┴───────────┴──────────────┴──────────────────────────────────────────────────────────────────────────────┘
 ```
 
-### Scan the Iceberg file
+### Scan the Iceberg table
+
+`LOAD FROM` is a Cypher query that scans a file or object element by element, but doesn’t actually
+move the data into a Kùzu table.
+
+To scan an iceberg table, you can do the following:
 
 ```cypher
 LOAD FROM 'kuzu/extension/iceberg/test/iceberg_tables/person_table' (file_format='iceberg', allow_moved_paths=true) RETURN *;
 ```
+
+Note: The `file_format` parameter is used to explicitly specify the file format of the given file instead of letting kuzu sniff the file format at runtime. When scanning from the Iceberg table, 
```

Comment:
```suggestion
Note: The `file_format` parameter is used to explicitly specify the file format of the given file instead of letting Kùzu autodetect the file format at runtime. When scanning from the Iceberg table, 
```